iam-login-service/src/main/java/it/infn/mw/iam/api/client/management/ClientManagementAPIController.java

```diff
@@ -157,6 +171,28 @@ public void disableClient(@PathVariable String clientId) {
     account.ifPresent(a -> managementService.updateClientStatus(clientId, false, a.getUuid()));
   }
 
+  @PatchMapping("/{clientId}/revoke-refresh-tokens")
+  @PreAuthorize("#iam.hasScope('iam:admin.write') or #iam.hasDashboardRole('ROLE_ADMIN')")
+  public void revokeRefreshTokens(@PathVariable String clientId) {
+    disableClient(clientId);
```

Comment:
I can understand disabling the client seems reasonable but in my opinion we could keep this operation separated. Reading the last comment of Hannah here https://github.com/indigo-iam/iam/issues/577#issuecomment-2245410776 I can understand that the flow is "I found a compromised client, I rotated the secret and I want to revoke all the issued tokens for a security reason". So probably you don't want to disable it. And I think this is not expected if I call an endpoint called "revoke-refresh-tokens". So, I'd remove this line.